src/test/java/HW20.java

```diff
@@ -0,0 +1,85 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class HW20 extends BaseTest {
+
+    @Test
+    public void DeletePlaylist() throws InterruptedException {
+        login("zahinedu@gmail.com", "te$t$tudent");
+        clickNewPlaylist();
+        createPlayList();
+        deletePlaylist();
+        playListDeleted();
+
+    }
+
+
+    public void createPlayList() {
+
+
+        WebElement newPlayList = driver.findElement(By.cssSelector(".fa-plus-circle"));
+        wait.until(ExpectedConditions.visibilityOf(newPlayList));
+        newPlayList.click();
+
+
+        WebElement addNewPlayList = driver.findElement(By.cssSelector("[data-testid='playlist-context-menu-create-simple']"));
+        wait.until(ExpectedConditions.visibilityOf(addNewPlayList));
+        addNewPlayList.click();
+
+
+        WebElement namePlayList = driver.findElement(By.xpath("//input[@name='name']"));
+        wait.until(ExpectedConditions.visibilityOf(namePlayList));
+        namePlayList.click();
+        namePlayList.clear();
+        namePlayList.sendKeys("Amira's List", Keys.ENTER);
+
+
+    }
+
+    public void deletePlaylist() {
+        WebElement deleteList = driver.findElement(By.xpath("//button[@title='Delete this playlist']"));
+        wait.until(ExpectedConditions.visibilityOf(deleteList));
+        deleteList.click();
+    }
+
+
+    public void playListDeleted() {
+        WebElement noPlayList = driver.findElement(By.xpath("//nav[@id='sidebar']"));
+        wait.until(ExpectedConditions.visibilityOf(noPlayList));
+        Assert.assertTrue(noPlayList.isDisplayed());
+    }
+
+        public void clickNewPlaylist() {
+           WebElement clickBtn = driver.findElement(By.xpath("//i[@title='Create a new playlist']"));
+            wait.until(ExpectedConditions.elementToBeClickable(By.xpath("//i[@title='Create a new playlist']")));
+              clickBtn.click();
+
+        }
+        }
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
**Please delete the empty line!** An Empty Line is a "Code Smell"
The short answer is, _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._